tests/test_execution.py

```diff
@@ -101,15 +101,10 @@ def test_federated_learning_workflow(global_execution_env):
 
 
 @pytest.mark.slow
-def test_tuples_execution_on_different_nodes(global_execution_env):
+def test_tuples_execution_on_different_nodes(factory, client_1, client_2,
+                                             initial_assets_1, initial_assets_2):
     """Execution of a traintuple on node 1 and the following testtuple on node 2."""
     # add test data samples / dataset / objective on node 1
-    factory, initial_assets, network = global_execution_env
```

Comment:
The factory obtained through global_execution_env made sure assets created in the tests and initial_assets would share uuid, with this change they will have 2 different uuid. I'm not sure about the impacts though, this needs investigating.

Reply:
Yes, at first sight I don't think it would be a problem. But as you said, it requires further investigation.

Reply:
@maeldebon what about this topic? Have you done some investigations?

Reply:
Ok, I investigated this.
There is indeed a difference between the two UUIDs, but the root stays the same.

**In the `factory` fixture:**
```
##### f._uuid #####
 c306ac2a6e7c4cc087e74b66f46f5b51_test_tuples_execution_on_same_node
```

**In the `data_envs` fixture:**
```
##### f._uuid #####
 c306ac2a6e7c4cc087e74b66f46f5b51_global
```